test/RPCSession.h

```diff
@@ -33,14 +37,21 @@ class IPCSocket: public boost::noncopyable
 public:
 	IPCSocket(std::string const& _path);
 	std::string sendRequest(std::string const& _req);
+#if defined(_WIN32)
```

Comment:
Please use a single base class that contains all shared data and logic and then two derived classes clearly separated by ifdefs.